lib/gems/pending/util/miq-password.rb

```diff
@@ -72,6 +73,40 @@ def self.encrypted?(str)
     !!split(str).first
   end
 
+  def self.encrypt_hash(values)
+    hash_values = values.deep_clone
+
+    hash_values.select { |f, _| f =~ /[Pp]assword/ }.each do |key, val|
```

Comment:
Regarding https://github.com/ManageIQ/manageiq-gems-pending/pull/63/files#r101402312 I think this regex should be put into a constant or method and used by both `encrypt_hash` and `decrypt_hash`.